R/f7Accordion.R

```diff
@@ -8,39 +8,7 @@
 #'
 #' @rdname accordion
 #'
-#' @examples
-#' # Accordion
-#' if (interactive()) {
-#'   library(shiny)
-#'   library(shinyMobile)
-#'
-#'   shinyApp(
-#'     ui = f7Page(
-#'       title = "Accordions",
-#'       f7SingleLayout(
-#'         navbar = f7Navbar("Accordions"),
-#'         f7Accordion(
-#'           id = "myaccordion1",
-#'           f7AccordionItem(
-#'             title = "Item 1",
-#'             f7Block("Item 1 content"),
-#'             open = TRUE
-#'           ),
-#'           f7AccordionItem(
-#'             title = "Item 2",
-#'             f7Block("Item 2 content")
-#'           )
-#'         )
-#'       )
-#'     ),
-#'     server = function(input, output, session) {
-#'       observe({
-#'         print(input$myaccordion1)
-#'       })
-#'     }
-#'   )
-#' }
-#'
+#' @example inst/examples/accordion/app.R
```

Comment:
@hypebright: I removed the plain code from the roxygen and pointed to the `inst` folder where examples are stored. That'll be a lot of examples but we can test apps with shinytest2.